packages/components/navbar/src/NavbarSwitcher/NavbarSwitcher.styles.ts

```diff
@@ -14,6 +14,19 @@ export const getNavbarSwitcherStyles = ({
   text: css({
     color: 'currentcolor',
     fontWeight: 'inherit',
+    lineHeight: 'unset',
+    'span:first-child': {
```

Comment:
It appears we are really only targeting the start `<span>` here.

Looking at the usage of `styles.text` from the condition @ L-88-97 - https://github.com/contentful/forma-36/pull/2934/files#diff-78e584a28d5563d6442791d3f7b1f32d1c9eae70a296d7f83293ca73c1810d8fR88-R97

If we don't have a `start`, then the `value` equals a string without `<span>` because we pass the `space` string to the `<SwitcherLabel value={space} />`.

If confirmed, let's move it to a new key for better understandability.